src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -102,14 +119,33 @@ export default {
 		const { loans, totalCount } = await runLoansQuery(
 			this.apollo,
 			{ ...this.flssLoanSearch, ...this.loanSearchState },
-			FLSS_ORIGIN_CATEGORY
+			FLSS_ORIGIN_LENDING_HOME
 		);
 		this.loans = loans;
 		this.totalCount = totalCount;
 		this.backupLoans = this.loans.slice(3);
 	},
+	computed: {
+		filterPageUrl() {
+			const location = this.flssLoanSearch.countryIsoCode?.toString();
+			const paramStr = JSON.parse(JSON.stringify({
```

Comment:
The JSON parse/stringify shouldn't be needed here. `URLSearchParams` can take an object directly.

Reply:
Sometimes some of the values are undefined, and using toString on URL Search Params seems to keep them there (i.e. `gender=undefined`) - the parsing removes the undefined values beforehand

Reply:
Gotcha. In that case, I would add a comment above this line, since I could see this parsing being removed accidentally without context in the future.